### DISCUSSION TITLE
marchId request

### DISCUSSION THREAD

**Discussion by splinedrive** [riscv/riscv-isa-manual#1225] [General]
I have a uLinux SoC (which has been published) and RV32IMA with SV32 (work in progress) but it runs well, it still needs to be optimized. I would like to have a MarchID for my cores and wanted to ask what requirements need to be met for that.

https://github.com/splinedrive/kianRiscV

**Comment by aswaterman** [comment]
I would say you should just go ahead and allocate an ID.
